spirv_reflect.c

```diff
@@ -1227,26 +1254,74 @@ static SpvReflectResult ParseFunction(
         CHECKED_READU32(p_parser, p_node->word_offset + 3,
                         p_func->callees[p_func->callee_count]);
         (++p_func->callee_count);
-      }
-      break;
-      case SpvOpLoad:
+      } break;
       case SpvOpAccessChain:
       case SpvOpInBoundsAccessChain:
       case SpvOpPtrAccessChain:
       case SpvOpArrayLength:
       case SpvOpGenericPtrMemSemantics:
-      case SpvOpInBoundsPtrAccessChain:
+      case SpvOpInBoundsPtrAccessChain: {
+        CHECKED_READU32(p_parser, p_node->word_offset + 3,
```

Comment:
Params should be all on one line or all on separate lines. Avoid paragraph style param packing. Same for the next few cases below.

Reply:
this was from clang-format